pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/errorprone/xml/CloseResource.xml

```diff
@@ -543,7 +543,7 @@ public class Bar {
 
     <test-code>
         <description>#1375 CloseResource not detected properly - ok</description>
-        <expected-problems>1</expected-problems>
+        <expected-problems>2</expected-problems>
```

Comment:
hm... doubling the violations here is weird. Do we report these cases twice?

Reply:
so, these are the messages:

n/a:6:	Ensure that resources like this ResultSet object are closed after use
n/a:11:	'rs' is not closed within a finally block, thus might not be closed at all in case of exceptions

they kind of mean the same thing: the resultset "rs" is maybe not closed after use...

Reply:
I'll add a check, so that we don't report violations twice for the same variable.